app/views/posts/new.html.erb

```diff
@@ -0,0 +1,24 @@
+
+<%= form_with model: [@user, @post], url: create_user_post_path  do |form| %>
+  <% if @post.errors.any? %>
+    <div>
+      <% @post.errors.full_messages.each do |message| %>
+        <div><%= message %></div>
+      <% end %>
+    </div>
+  <% end %>
+
+  <div class="">
+    <%= form.label :title %>
+    <%= form.text_field :title, placeholder: "Add post title" %>
+  </div>
+  <div class="">
+    <%= form.label :text %>
+    <%= form.text_field :text, placeholder: "Add post body" %>
+  </div>
+  <div class="">
+    <%= form.submit "Add post" %>
+  </div>
+
+<% end %>
```

Comment:
- [ ] I'm not able to create a post because an error shows up on the posts/new page. Please ensure the current user is able to create a post, comment, and like.
<img width="473" alt="image" src="https://user-images.githubusercontent.com/77190203/179350881-d3657b9d-1a93-4b18-931c-4ca3471798f4.png">